docs/ocean/design_docs/vert-lagrang-remap/vert-lagrang-remap.rst

```diff
@@ -0,0 +1,524 @@
+.. role:: red
```

Comment:
@cbegeman, out of curiosity, what does this do?

Reply:
That's leftover from my personal markup. I'll be removing it. If you add these lines at the top or in a stylesheet (building with rst2html) then you can use `:red:` to get red text:

`.. raw:: html`
` `
`    <style> .red {color:red} </style>`

Reply:
Ah, good to know.